### PR TITLE
Feature: admin user added

### PR DESCRIPTION
**Pull request by ivanfelfo**
Feature:
- created admin_users table
- admin_user login
Screen Shot 2021-05-14 at 16 55 46
Screen Shot 2021-05-14 at 16 56 18

### CODE REVIEW

File: db/seeds.rb [hunk 1]
```diff
@@ -7,3 +7,4 @@
 #
 #   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
 #   Character.create(name: 'Luke', movie: movies.first)
+AdminUser.create!(email: 'admin@example.com', password: 'password', password_confirmation: 'password') if Rails.env.development?
```
Comment (martinjaimem) [inline db/seeds.rb:10]:
```suggestion
AdminUser.create!(email: 'admin@example.com', password: 'password', password_confirmation: 'password') if Rails.env.development?

```

File: app/models/admin_user.rb [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+  devise :database_authenticatable, 
+         :recoverable, :rememberable, :validatable
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
+  devise :database_authenticatable, 
+         :recoverable, :rememberable, :validatable
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
+  devise :database_authenticatable, 
+         :recoverable, :rememberable, :validatable
```
Comment (martinjaimem) [inline app/models/admin_user.rb:13]:
```suggestion
```

File: Gemfile [hunk 3]
```diff
@@ -12,10 +12,11 @@ gem 'pg', '~> 1.1'
 # Use Puma as the app server
 gem 'puma', '~> 5.0'
 
+gem 'activeadmin'
```
Comment (juanmferreira93) [inline Gemfile:15]: please add the gem version here

File: app/admin/dashboard.rb [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+    #       end
+    #     end
+    #   end
+
+    #   column do
+    #     panel "Info" do
+    #       para "Welcome to ActiveAdmin."
+    #     end
+    #   end
+    # end
```
Comment (juanmferreira93) [inline app/admin/dashboard.rb:30]: delete this comments lines if you don't need it

File: config/application.rb [hunk 5]
```diff
@@ -12,6 +12,7 @@
 require 'action_text/engine'
 require 'action_view/railtie'
 require 'action_cable/engine'
+require 'sprockets/railtie'
 # require "sprockets/railtie"
```
Comment (juanmferreira93) [inline config/application.rb:16]: 👀

File: app/models/admin_user.rb (hunk 2 above)
Comment (ivanfelfo) [reply app/models/admin_user.rb]: 👍

File: Gemfile (hunk 3 above)
Comment (ivanfelfo) [reply Gemfile:15]: 👍

File: app/admin/dashboard.rb (hunk 4 above)
Comment (ivanfelfo) [reply app/admin/dashboard.rb:30]: 👍

File: config/application.rb (hunk 5 above)
Comment (ivanfelfo) [reply config/application.rb:16]: 👍

**Review by juanmferreira93 — APPROVED** [review]

**Review by martinjaimem — APPROVED** [review]

**Pull request merged**